local.bib

```diff
@@ -28,3 +28,199 @@ @inproceedings{10.1117/12.926797
+         {Aleksi{\'c}}, J. and {Allsman}, R.~A. and {AlSayyad}, Y. and
+         {Alt}, J. and {Armstrong}, R. and {Basney}, J. and {Becker}, A.~C. and
+         {Becla}, J. and {Biswas}, R. and {Bosch}, J. and {Boutigny}, D. and
+         {Kind}, M.~C. and {Ciardi}, D.~R. and {Connolly}, A.~J. and
+         {Daniel}, S.~F. and {Daues}, G.~E. and {Economou}, F. and
+         {Chiang}, H. -F. and {Fausti}, A. and {Fisher-Levine}, M. and
+         {Freemon}, D.~M. and {Gris}, P. and {Hernandez}, F. and {Hoblitt}, J. and
+         {Ivezi{\'c}}, Z. and {Jammes}, F. and {Jevremovi{\'c}}, D. and
+         {Jones}, R.~L. and {Kalmbach}, J.~B. and {Kasliwal}, V.~P. and
+         {Krughoff}, K.~S. and {Lurie}, J. and {Lust}, N.~B. and
+         {MacArthur}, L.~A. and {Melchior}, P. and {Moeyens}, J. and
+         {Nidever}, D.~L. and {Owen}, R. and {Parejko}, J.~K. and
+         {Peterson}, J.~M. and {Petravick}, D. and {Pietrowicz}, S.~R. and
+         {Price}, P.~A. and {Reiss}, D.~J. and {Shaw}, R.~A. and {Sick}, J. and
+         {Slater}, C.~T. and {Strauss}, M.~A. and {Sullivan}, I.~S. and
+         {Swinbank}, J.~D. and {Van Dyk}, S. and {Vuj{\v{c}}i{\'c}}, V. and
+         {Withers}, A. and {Yoachim}, P.},
+        title = "{The LSST Data Management System}",
+     keywords = {Astrophysics - Instrumentation and Methods for Astrophysics},
+    booktitle = {Astronomical Data Analysis Software and Systems XXV},
+         year = "2017",
+       editor = {{Lorente}, N.~P.~F. and {Shortridge}, K. and {Wayth}, R.},
+       series = {ASP Conf.\ Ser.},
+       volume = {512},
+        month = "Dec",
+        pages = {279},
+archivePrefix = {arXiv},
+       eprint = {1512.07914},
+ primaryClass = {astro-ph.IM},
+       adsurl = {https://ui.adsabs.harvard.edu/abs/2017ASPC..512..279J},
+      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+
+@misc{LSE-61,
+author =       {Gregory Dubois-Felsmann and Tim Jenness},
+title =        "{LSST Data Management Subsystem Requirements, v8.4}",
+howpublished=  "{LSE-61, Rubin Observatory, \url{https://ls.st/LSE-61}}",
+year =         2020,
+}
+
+@misc{LPM-17,
+author =       {{\v Z}. {Ivezi{\'c}} and {The LSST Science
+                Collaboration}},
+title =        "{LSST Science Requirements Document, v5.2.4}",
+year =         2018,
+howpublished = {LPM-17, Rubin Observatory, \url{https://ls.st/LPM-17}},
+}
+
+@misc{SQR-006,
+author =       {Jonathan Sick},
+title =        "{The LSST the Docs Platform for Continuous Documentation Delivery}",
+year =         2016,
+howpublished = {SQR-006, Rubin Observatory, \url{https://sqr-006.lsst.io}},
+}
+
+@misc{LSE-68,
+author =       {Gregory Dubois-Felsmann},
+title =        "{Camera Data Acquisition Interface, v2.4}",
+year =         2020,
+howpublished = {LSE-68, Rubin Observatory, \url{https://ls.st/LSE-68}},
+}
+
+@ARTICLE{2019ApJ...873..111I,
+       author = {{Ivezi{\'c}}, {\v{Z}}eljko and {Kahn}, Steven M. and
```

Comment:
probably want this to be "FirstAuthor and others" to prevent lines and lines of names in the references.